### ISSUE TITLE
error: Could not find the FFTW header 'fftw3.h'

### ISSUE THREAD

**Issue by mshane87** [pyFFTW/pyFFTW#289]
The fftw3.h file is already included in 'include\win' but still I am getting this error. Please help

![image](https://user-images.githubusercontent.com/34945617/80592615-5b712000-8a3d-11ea-8c91-487b3a5bf8a4.png)

**Comment by hgomersall** [comment]
Windows is a pain. Is there a reason why you want to build yourself? 0.12 is pretty up to date and is available on either pypi or bintray.

**Comment by mshane87** [comment]
Actually, I had installed pyFFTW 0.12 and it was successful but when I used it to build pyGFT project, it is giving error: could not find the file "fftw3.lib". So I thought pyFFTW3 was required.

![image](https://user-images.githubusercontent.com/34945617/80595655-bbb69080-8a42-11ea-9a23-1b7f6f92b77b.png)

**Comment by hgomersall** [comment]
I'm pretty sure this has nothing to do with pyfftw. I suggest asking the pyGFT people for help on building. Closing until there is something specific we can address.

**Comment by mshane87** [comment]
ok thanks henry

**Comment by gstr001** [comment]
So, I try to install pyfftw by pip but it cannot work. Can anybody here help me with this issue?

**Comment by hgomersall** [comment]
> So, I try to install pyfftw by pip but it cannot work. Can anybody here help me with this issue?

Is your problem related to this issue? Are you trying on Python 3.9?

**Comment by gstr001** [comment]
> > So, I try to install pyfftw by pip but it cannot work. Can anybody here help me with this issue?
> 
> Is your problem related to this issue? Are you trying on Python 3.9?

Yes, I try to install pyfftw on my windows sys. I have downloaded it from GitHub and tried to run the setup.py. It is said could not find the fftw header 'fftw3.h'. If I used pip to install it, there is also an error. 
![Capture](https://user-images.githubusercontent.com/36036607/106628813-2f447b80-6548-11eb-83de-9af24a30493e.PNG)

**Comment by hgomersall** [comment]
Assuming you have FFTW installed, I suggest either using Python 3.8, waiting for pyFFTW to release for 3.9, or working out how to fix your compiler problems. Beyond that I'm not able to help, apologies.

**Comment by gstr001** [comment]
> Assuming you have FFTW installed, I suggest either using Python 3.8, waiting for pyFFTW to release for 3.9, or working out how to fix your compiler problems. Beyond that I'm not able to help, apologies.

OK, I will go back to v3.8 and try again. This issue has really annoyed me. Thank you!

**Comment by robmarkcole** [comment]
Also hitting this error on M1 Mac (Arm)

**Comment by markuskreitzer** [comment]
I'm hitting an error on Mac M1 Pro (ARM 64 bit v7) as well.

```
      DEBUG:__main__:Checking with includes ['fftw3.h']...no
      error: Could not find the FFTW header 'fftw3.h'
      [end of output]
```
